diff --git a/node_modules/react-native-audio-pro/android/build.gradle b/node_modules/react-native-audio-pro/android/build.gradle
index afeb518..fa76270 100644
--- a/node_modules/react-native-audio-pro/android/build.gradle
+++ b/node_modules/react-native-audio-pro/android/build.gradle
@@ -78,10 +78,11 @@ dependencies {
 	implementation "com.facebook.react:react-android"
 	implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
 
-	implementation "androidx.media3:media3-exoplayer:1.6.0"
-	implementation "androidx.media3:media3-session:1.6.0"
-	implementation "androidx.media3:media3-common:1.6.0"
-	implementation "androidx.media3:media3-common-ktx:1.6.0"
+	// Align media3 version with expo-video and expo-audio
+	implementation "androidx.media3:media3-exoplayer:1.8.0"
+	implementation "androidx.media3:media3-session:1.8.0"
+	implementation "androidx.media3:media3-common:1.8.0"
+	implementation "androidx.media3:media3-common-ktx:1.8.0"
 
 	implementation "androidx.core:core-ktx:1.16.0"
 	implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.8.7"
diff --git a/node_modules/react-native-audio-pro/lib/module/useAudioPro.js b/node_modules/react-native-audio-pro/lib/module/useAudioPro.js
index ca0cef5..a8e28f2 100644
--- a/node_modules/react-native-audio-pro/lib/module/useAudioPro.js
+++ b/node_modules/react-native-audio-pro/lib/module/useAudioPro.js
@@ -11,7 +11,6 @@ const selectAll = state => ({
   volume: state.volume,
   error: state.error
 });
-const select = internalStore;
 
 /**
  * React hook for accessing the current state of the audio player.
@@ -21,11 +20,7 @@ const select = internalStore;
  * subscribe only to the specific piece of state your component needs.
  */
 
-export function useAudioPro(selector, equalityFn) {
-  const selectAllWithShallow = useShallow(selectAll);
-  if (selector) {
-    return select(selector, equalityFn);
-  }
-  return internalStore(selectAllWithShallow);
+export function useAudioPro(selector) {
+  return internalStore(useShallow(selector ?? selectAll));
 }
 //# sourceMappingURL=useAudioPro.js.map
\ No newline at end of file
